js/georgia-main.js

```diff
@@ -2645,7 +2660,7 @@ function createButtonObjects(ww, wh) {
 		var y = pref.show_transport_below ? wh - geo.lower_bar_h - scaleForDisplay(10) - buttonSize : scaleForDisplay(10);
 		var w = buttonSize;
 		var h = w;
-		var p = scaleForDisplay(5); // space between buttons
+		var p = scaleForDisplay(pref.transport_buttons_spacing); // space between buttons

```

Comment:
I didn't examine this code too closely when I added transport scaling. I thought the space between was proportional to the button size, and feel like it probably ought to be. I'll have to think about this.

Reply:
I personally use 25, which might seem massive. I can't tell you why, just a personal preference. It doesn't seem to negatively affect anything though.